notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb

```diff
@@ -1170,11 +1106,11 @@
         "id": "e1d089d1d0d9"
       },
       "source": [
-        "### Get the latest state of the production endpoint\n",
+        "## Get the latest state of the production endpoint\n",
         "\n",
-        "Now that the pipeline has finished, the challenger (version 2) model has replaced the previous blessed model on the production endpoint.\n",
+        "Once your pipeline execution is finished, you can notice that the challenger (version 2) model has replaced the previous blessed model on the production endpoint.\n",
         "\n",
-        "Next your display the latest information on the deployed models for the production endpoint, and then display the traffic split. The resource ID for the 100% entry is the resource ID for the challenger (v2) model."
+        "Now, display the latest details of the deployed models from the production endpoint, and then display the traffic split. Notice that the resource ID for the 100% entry is the resource ID for the challenger (v2) model."
```

Comment:
change: " Notice that the resource ID" 

to:  "Notice the resource ID" (remove extra "that")